src/EditorFeatures/CSharpTest/CodeRefactorings/AddMissingImports/CSharpAddMissingImportsRefactoringProviderTests.cs

```diff
@@ -336,5 +336,51 @@ public class D { }
 
             await TestMissingInRegularAndScriptAsync(code);
         }
+
+        [WorkItem(31768, "https://github.com/dotnet/roslyn/issues/31768")]
+        [WpfFact]
+        public async Task AddMissingImports_AddMultipleImports_NoPreviousImports()
+        {
+            var code = @"
+class C
+{
+    [|public D Foo { get; }
+    public E Bar { get; }|]
+}
+
+namespace A
+{
+    public class D { }
+}
+
+namespace B
+{
+    public class E { }
+}
+";
+
+            var expected = @"
+using A;
```

Comment:
this newline seems undesirable.

Reply:
The logic of the AddImport code fix is to move any leading trivia to the new import statement.

Reply:
What is the proposed fix? I think it would probably need to be addressed separate from this PR.

Reply:
A using goes at the top of the file, unless there is a banner (i.e. copyright) above it. In which case, it goes after a separate blank line after that banner.

Reply:
I'm fine with the behavior presented here (in this test). If the user didn't want the file to start with a blank line, then the input code would not start with a blank line. By starting with a blank line, the premise for this change request is invalidated.